concepts/05 Widgets/DataGrid/20 Editing/10 User Interaction/User Interaction.md

```diff
@@ -84,6 +145,54 @@ With the **DataGrid** you can edit data in several modes. Use the **editing**.[m
         ],
         // ...
     })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ... >
+            <DxEditing mode="row" /> <!-- 'batch' | 'cell' | 'form' | 'popup' -->
```

Comment:
Editing does not work without allowUpdating/allowAdding/allowDeleting. So, it is a synthetic, unworkable example. It will be better add allowUpdating to it.

Reply:
I've merged this and the examples in which we set the `allow...` propeties.